README.md

```diff
@@ -1,17 +1,11 @@
-# Things
+## Original mod
+  * Original mod: https://www.curseforge.com/minecraft/mc-mods/things-fabric
```

Comment:
You might need to rebase this, to keep the original

Reply:
Done